go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/metal3-io/baremetal-operator
 
-go 1.21
+go 1.21.6
```

Comment:
Any specific reason to pin it to patch version?

Reply:
This is apparently the new way Go does it starting with 1.21. I haven't done this change - `go mod tidy` does it (based on Gophercloud's go.mod, I believe).

Reply:
Interesting , I didnt face it when I bumped it to go 1.21, I have to check it again.